src/http.ts

```diff
@@ -187,15 +187,15 @@ class Http {
 
       xhr.onerror = function () {
         if (host && (host === 'http://localhost:52516' || host === 'https://radar-verify.com:52516')) {
-          reject(new RadarVerifyAppError());
+          reject(navigator.onLine ? new RadarVerifyAppError() : new RadarNetworkError());
```

Comment:
Currently `GET localhost:52516/verify` returns `RadarVerifyAppError` both when the Radar Verify app isn't running AND when the device is offline. This is a quick fix to return `RadarNetworkError` in the latter case instead